### PR TITLE
AddingReduxOnLogin

### PR DESCRIPTION
**Pull request by K0unna00**

### CODE REVIEW

File: src/Pages/Header/Header.tsx [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
+          <Link to="/">Home</Link>
           <a href="/#">About Us</a>
           <a href="/#">Our Work</a>
           <a href="/#">Clients</a>
-          <Link to="/our-blog">
-            <a href="/#">Our Blog</a>
-          </Link>
-          <a href="/#">Profile</a>
+          <Link to="/our-blog">Our Blog</Link>
+          <Link to="/login">{login.isSigned ? 'Bextiyar' : 'Profile'}</Link>
```
Comment (Negodyaj) [inline src/Pages/Header/Header.tsx]: I think it should be like this:
{login.isSigned ? <Link to="/profile">Profile</Link> : <Link to="/login">Log in</Link>}

File: src/Pages/LoginPage/LoginPage.tsx [hunk 2]
```diff
@@ -1,14 +1,23 @@
 import './LoginPage.scss';
+import { Link } from 'react-router-dom';
+import { loginSlice } from './redux/loginPageSlice';
+import { useDispatch } from 'react-redux';
 
 export const LoginPage = () => {
+  const dispatch = useDispatch();
+
+  const onLoginPress = (e: any) => {
```
Comment (Negodyaj) [inline src/Pages/LoginPage/LoginPage.tsx]: you must not use "any" =)
if you don't use event's object, you can leave these brackets empty

File: src/modules/redux/store.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import { loginSlice } from '../../Pages/LoginPage/redux/loginPageSlice';
+
+export const store = configureStore({
+  reducer: {
+    login: loginSlice.reducer,
+  },
+});
+
+export type RootState = ReturnType<typeof store.getState>;
```
Comment (Negodyaj) [inline src/modules/redux/store.ts:10]: why do you need "modules" folder?..

Comment (K0unna00) [reply src/modules/redux/store.ts:10]: Javid told us

File: src/Pages/Header/Header.tsx (hunk 1 above)
Comment (K0unna00) [reply src/Pages/Header/Header.tsx]: fixed

File: src/Pages/LoginPage/LoginPage.tsx (hunk 2 above)
Comment (K0unna00) [reply src/Pages/LoginPage/LoginPage.tsx]: fixed

File: src/modules/redux/store.ts (hunk 3 above)
Comment (Negodyaj) [reply src/modules/redux/store.ts:10]: ok, let it be like this then)

**Pull request merged**